### PR TITLE
Add files for FreeBSD-UPB 2019q2 status report

### PR DESCRIPTION
**Pull request by elena19m**
Signed-off-by: Elena Mihailescu <elenamihailescu22@gmail.com>

### CODE REVIEW

File: 2019q2/bhyve_snapshot.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Link:	[Github wiki - How to Save and Restore a bhyve guest](https://github.com/FreeBSD-UPB/freebsd/wiki/Save-and-Restore-a-virtual-machine-using-bhyve)
+
+Link:	[Github wiki - Suspend/resume test matrix](https://github.com/FreeBSD-UPB/freebsd/wiki/Suspend-Resume-test-matrix)
+
+Link:	[Phabricator review - bhyve Snapshot Save and Restore](https://reviews.freebsd.org/D19495)
+
+The Save/Restore for bhyve feature is a suspend and resume facility added to the
+FreeBSD/amd64's hypervisor, bhyve. The bhyvectl tool is used to save the guest
+state in three files (a file for the guest memory, a file for devices' and CPU's
```
Comment (debdrup) [inline 2019q2/bhyve_snapshot.md]: s/devices\'/devices/
s/CPU\'s/CPUs/

File: 2019q2/bhyve_snapshot.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+state and another one for some metadata that are used in the restore process).
```
Comment (debdrup) [inline 2019q2/bhyve_snapshot.md]: s/state/states

File: 2019q2/bhyve_snapshot.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+   * Open ticket on Phabricator;
+   * Apply feedback received from community
+
+Future tasks:
+
+   * Add suspend/resume support for nvme;
+   * Add suspend/resume support for virtio-console;
+   * Add suspend/resume support for virtio-scsi;
+   * Add TSC offseting for restore for AMD CPUs;
```
Comment (debdrup) [inline 2019q2/bhyve_snapshot.md]: s/offseting/offsetting/

**Comment by elena19m** [comment]
> Just a few small issues that need fixing. :)

Thank you for your feedback. I've addressed the comments.

**Pull request merged**